scripts/deploy_assisted_service.sh

```diff
@@ -58,16 +59,17 @@ fi
 mkdir -p build
 
 if [ "${OPENSHIFT_INSTALL_RELEASE_IMAGE}" != "" ]; then
-    export RELEASE_IMAGES=$(skipper run ./scripts/override_release_images.py --src ./assisted-service/data/default_release_images.json)
-    if [ -z "${OS_IMAGES:-}" ]; then
-        export OS_IMAGES=$(skipper run ./scripts/override_os_images.py --src ./assisted-service/data/default_os_images.json)
-    fi
+    RELEASE_IMAGES=$(skipper run ./scripts/override_release_images.py --src ./assisted-service/data/default_release_images.json)
+    export RELEASE_IMAGES
```

Comment:
this separation helps errors to be raised here instead of the code just continuing and having an empty value in ``RELEASE_IMAGES/OS_IMAGES``